### PR TITLE
testing: add e2e tests for rolebindings

### PR DESCRIPTION
**Pull request by dvirgilad**

### CODE REVIEW

**Comment by dvirgilad** [comment]
fixes #92

File: test/e2e_tests/rolebindings_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+		By("Creating a child namespace")
+		nsChild := GenerateE2EName("child", testPrefix, randPrefix)
+		CreateSubnamespace(nsChild, nsRoot, randPrefix, false, storage, "50Gi", cpu, "50", memory, "50Gi", pods, "50", gpu, "50")
+
+		By("Checking that the rolebinding has been created in the child namespace")
+		FieldShouldContain("rolebinding", nsChild, "test-admin-"+user+"-"+nsRoot, ".metadata.name", "test-admin-"+user+"-"+nsRoot)
+
+		By("Deleting the rolebinding from the root namespace")
+		MustRun("kubectl delete rolebinding test-admin-" + user + "-" + nsRoot + " -n " + nsRoot)
```
Comment (mzeevi) [inline test/e2e_tests/rolebindings_test.go:55]: Wrap this in a function like in other test cases

**Review by mzeevi — CHANGES_REQUESTED** [review]
Looks fine this is missing some test cases, go over the controllers and validators to see what you've missed

**Review by mzeevi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**